packages/yourloops/lib/auth/user.ts

```diff
@@ -136,6 +157,10 @@ export default class User {
     return !this.isFirstLogin() && this.shouldRenewConsent()
   }
 
+  hasToDisplayTrainingInfoPage(): boolean {
+    return this.isFirstLogin() || this.newTrainingAvailable()
```

Comment:
Is it really necessary to have the condition `this.isFirstLogin()`?

Reply:
Yes because it should be displayed when it's a first login ...